packages/deployment-service/src/utils/project-dir.ts

```diff
@@ -4,6 +4,9 @@ import { PipelineEntity } from './../entities'
 export const projectDir = (dir: string, pipeline: PipelineEntity) =>
   resolve(dir, `${(pipeline.repository as string).split('/').pop()}-master`)
 
-export const developerDir = (pipeline: PipelineEntity): string => `/mnt/efs1/project/${pipeline.developerId}/`
+const startDir = 'mnt/efs1'
+// const startDir = 'tmp'
```

Comment:
needed still?

Reply:
Sort of, use /tmp for local testing. Quite annoying but don't really want to add an env for this just yet